example/spec/parallel/discoverGranules/DiscoverGranulesS3SuccessSpec.js

```diff
@@ -98,9 +106,15 @@ describe('The DiscoverGranules workflow', () => {
   });
 
   afterAll(async () => {
-    await deleteGranule({ prefix: stackName, granuleId: expectedGranuleId });
+    await Promise.all(discoverGranulesOutput.payload.granules.map(
```

Comment:
I'm not sure I like the situation with this test in that there is nowhere *checking* the database status prior to these deletions (getExecutionStatus uses the Step Function interface only) however that's not *directly* relevant to this PR.